composer.json

```diff
@@ -19,7 +19,7 @@
         "tecnickcom/tcpdf": "^6.3"
     },
     "require-dev": {
-        "phpunit/phpunit": "^8.5",
+        "phpunit/phpunit": "^5.0 || ^9.0",
```

Comment:
🟠  Are we sure this shouldn't just be `^9.0`? I know this came from the main repo but our fork didn't have v5.0 support before